integration-tests/jpa-postgresql/src/test/java/io/quarkus/it/jpa/postgresql/JPAFunctionalityInGraalITCase.java

```diff
@@ -21,9 +21,14 @@ public void verifyJdkXmlParsersHavebeenEcludedFromNative() {
         report.assertContains(org.postgresql.jdbc.PgSQLXML.class);
         report.assertContains(UUIDJdbcType.class);
 
-        //And finally verify we managed to exclude the JDK XML because of having hinted the analysis
-        //(See io.quarkus.jdbc.postgresql.runtime.graal.SQLXLMFeature )
+        // And finally verify we exclude "com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl" which is
+        // the fallback implementation class name used in javax.xml.transform.TransformerFactory.newInstance()
+        // whose invocation gets triggered when io.quarkus.jdbc.postgresql.runtime.graal.SQLXLMFeature is enabled.
+        // We cannot only use class javax.xml.transform.TransformerFactory directly since delegation to
+        // the implementation might get inlined, thus resulting in 'javax.xml.transform.TransformerFactory'
+        // not showing up as a used class in the reports (due to '-H:+InlineBeforeAnalysis').
         report.assertContainsNot(javax.xml.transform.TransformerFactory.class);
```

Comment:
If `javax.xml.transform.TransformerFactory` never appears in the report anyway, maybe we should remove this? Otherwise it's a bit confusing...

Reply:
The reason I kept it is because the only reason it doesn't appear in the report when rynning `withxml` is because it gets optimized away. It should definitely keep not appearing in this test, so I think it remains a valid assertion.